src/cim/components/common/ValidatedConditionAlert.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+import { StatusCondition } from '../../types';
+
+type ValidationAlertProps = {
+  condition: StatusCondition<'Validated'>;
+};
+
+const ValidatedConditionAlert = ({ condition }: ValidationAlertProps) => {
+  const { status, message } = condition;
+  if (status === 'True') return null;
+  if (status === 'Unknown') {
+    return <Alert variant={AlertVariant.info} title={message} isInline />;
+  }
+  // const [title, ...messages] = message.split(/[:,]/);
+  const [title, ...messages] = message.split(/:|\.,/);
```

Comment:
A highway to hell.

Reply:
I agree, I am not sure if there is a better way to parse the messages right now. There is an open Jira requesting support for validations_info data structure from the backend.